src/Build/BackEnd/Components/SdkResolution/SdkResolverService.cs

```diff
@@ -355,6 +355,8 @@ private SdkResult ResolveSdkUsingAllResolvers(int submissionId, SdkReference sdk
 
                 if (result.Success)
                 {
+                    loggingContext.LogComment(MessageImportance.Low, "SucceededToResolveSDK", sdk.ToString());
```

Comment:
I believe the log line should include information from the `result` as it may not be obvious how exactly it was resolved given the `sdk` (the request).

Reply:
Done. I included info on the sdk resolver that resolved the sdk and some info on the result. These properties are asserted in the tests, that is why I included them: `Success` (the message already says that it is a success), `SdkReference` (the resolved sdk in the message), `Warnings` (are logged after the success message), `Path` and `Version`(added in the message).